src/rouge/core/workflow/status.py

```
@@ -23,3 +23,48 @@ def update_status(issue_id: int, status: str) -> None:
+
+    This is a best-effort operation - database failures are logged but never halt
+    workflow execution. Successful updates are logged at DEBUG level, failures
+    at ERROR level.
+
+    Args:
+        issue_id: The Rouge issue ID
+    """
+    try:
+        client = get_client()
+        client.table("issues").update({"status": "patch pending"}).eq("id", issue_id).execute()
+        logger.debug(f"Issue {issue_id} status updated to 'patch pending'")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)